src/foam/dao/Relationship.js

```diff
@@ -182,6 +182,14 @@ foam.CLASS({
         Only used for many to many relationships.
         Name of the class that this junction class should inherit from.
       `
+    },
+    {
+      class: 'StringArray',
+      name: 'ids',
+      documentation: "Ids of the relationship.",
+      factory: function() {
```

Comment:
Doesn't just using value: work?

Reply:
value works.

Reply:
Well, value doesn't work. StringArray property has a default factory and factory is evaluated before value.